packages/sanity/src/structure/panes/document/documentPanel/header/perspective/__tests__/DocumentPerspectiveMenu.test.tsx

```diff
@@ -90,45 +101,33 @@ describe('DocumentPerspectiveMenu', () => {
     const wrapper = await createTestProvider()
     render(<DocumentPerspectiveMenu />, {wrapper})
 
-    expect(screen.getByTestId('button-document-release')).toBeInTheDocument()
+    const linkButton = screen.getByRole('link', {name: 'Spring Drop'})
+    expect(linkButton).toBeInTheDocument()
+    expect(linkButton).toHaveAttribute('href', '/intent/release/id=spring-drop')
+    expect(linkButton).toHaveTextContent('Spring Drop')
   })
 
   it('should not render the bundle badge if the document does not exist in the bundle', async () => {
-    mockUseDocumentPane.mockReturnValue({
-      existsInBundle: false,
-    })
-
-    const wrapper = await createTestProvider()
-    render(<DocumentPerspectiveMenu />, {wrapper})
-
-    expect(screen.queryByTestId('button-document-release')).toBeNull()
-  })
-
-  it('should navigate to the release intent when the bundle badge is clicked', async () => {
```

Comment:
This test doesn't apply anymore, because we are not rendering a button, we are now rendering a `<a>` 
That's also the reason why instead of using a `getByTestId` getter for the button we are now using a `getByRole("link")` getter in this tests